src/text.css.ts

```diff
@@ -17,15 +19,8 @@ export const vars = {
 
 export const withWordBreak = style({
     overflowWrap: 'anywhere',
-    '@supports': {
-        'not (overflow-wrap: anywhere)': {
-            // "overflow-wrap: anywhere" is not supported in Safari
-            // "word-break: break-word" has the same effect as "word-break: normal" and "overflow-wrap: anywhere",
-            // regardless of the actual value of the overflow-wrap property.
-            wordBreak: 'break-word',
-        },
-    },
-});
+    wordBreak: ['keep-all', 'auto-phrase'],
+} as ComplexStyleRule);
```

Comment:
just curious, why is this type cast needed? And why wasn't needed before?

Reply:
Because we are now using auto-phrase, an experimental property, and TS complains without the type cast.

More context here: https://github.com/vanilla-extract-css/vanilla-extract/discussions/740#discussioncomment-3199094